Content.Shared/_RMC14/Dropship/Weapon/SharedDropshipWeaponSystem.cs

```diff
@@ -54,10 +56,12 @@ public abstract class SharedDropshipWeaponSystem : EntitySystem
     [Dependency] private readonly IGameTiming _timing = default!;
     [Dependency] private readonly SharedTransformSystem _transform = default!;
     [Dependency] private readonly SharedUserInterfaceSystem _ui = default!;
+    [Dependency] private readonly UseDelaySystem _useDelay = default!;
+    [Dependency] private readonly EntityManager _entityManager = default!;
 
     private static readonly EntProtoId DropshipTargetMarker = "RMCLaserDropshipTarget";
 
-    private bool _casDebug;
+    public bool CasDebug;
```

Comment:
You can do this instead to make sure that nothing else modifies this value
```suggestion
    public bool CasDebug { set; private set; }
```

Reply:
Yes sorry, `get; private set;`, I wrote this on my phone originally